posts/flat-file-exports/flat-files-compare.qmd

```diff
@@ -0,0 +1,440 @@
+  2        4255       29791   0.5119157    3.583410
+  3        8511       29791   1.0238314    3.583410
+  4       12767       29791   1.5357471    3.583410
+  5       17023       29791   2.0476628    3.583410
+  6       21279       29791   2.5595785    3.583410
+  7       25535       29791   3.0714942    3.583410
+  8       29791       29791   3.5834099    3.583410
+  9           0       25535   0.0000000    3.071494
+  10        4255       25535   0.5119157    3.071494
+```
+
+:::
+
+# Expression Matrix
+
+For expression matrices, we see that NegPrb<\\d+> is changed to Negative<\\d+> and
+the newer format has SystemControls.
+
+::: {.panel-tabset group="version"}
+
+## Legacy
+
+```{r}
+#| eval: false
+library(plyr)
+library(dplyr)
+expr_file_old <- read.csv("path/to/NSCLC/Lung5_Rep1/Lung5_Rep1-Flat_files_and_images/Lung5_Rep1_exprMat_file.csv", header=TRUE)
+
+expr_file_old %>% select(fov:AATK, ZFP36:NegPrb3, NegPrb23) %>% head(10)
+
+```
+
+```
+  fov cell_ID AATK ... ZFP36 NegPrb3 ... NegPrb23
+  1       0   15 ...   184      12 ...       13
+  1       1    0 ...     0       0 ...        0
+  1       2    0 ...     1       0 ...        0
+  1       3    0 ...     0       0 ...        0
+  1       4    0 ...     0       0 ...        0
+  1       5    0 ...     0       0 ...        0
+  1       6    0 ...     0       0 ...        0
+  1       7    0 ...     1       0 ...        0
+  1       8    0 ...     0       0 ...        0
+  1       9    0 ...     0       0 ...        0
+```
+
+## AtoMx 1.3.2
+
+```{r}
+#| eval: false
+
+expr_file_new <- read.csv("path/to/breast_cancer_example/sample_dir_formatted/AUG29_13INTEGR_6K_BRST_PS_S2_exprMat_file.csv.gz", header=TRUE)
+
+expr_file_new %>% select(fov:A1BG, ZZZ3:Negative1, Negative9:SystemControl1, SystemControl99) %>% head(10)
+
+```
+
+```
+  fov cell_ID A1BG ... ZZZ3 Negative1 ... Negative9 SystemControl1 ... SystemControl99
+  1       1    0 ...    0         0 ...         0              0 ...               0
+  1       2    0 ...    0         0 ...         0              0 ...               0
+  1       3    0 ...    0         0 ...         0              0 ...               0
+  1       4    1 ...    0         0 ...         0              0 ...               0
+  1       5    0 ...    0         0 ...         0              0 ...               0
+  1       6    0 ...    0         0 ...         0              0 ...               0
+  1       7    0 ...    0         0 ...         0              0 ...               0
+  1       8    0 ...    0         0 ...         0              0 ...               0
+  1       9    1 ...    0         0 ...         0              0 ...               0
+  1      10    0 ...    0         0 ...         0              0 ...               0
+```
+
+:::
+
+# Metadata file
+
+The primary difference between legacy and AtoMx formats for the metadata is that the latter includes processed results (_i.e._, extra columns).
```

Comment:
I would also call out the area in microns vs pixels.

Reply:
I have refactored this post: 1a0416d378b30a2f45278643629781e3b497fd92